compiler/cpp/src/thrift/generate/t_swift_3_generator.cc

```diff
@@ -0,0 +1,2553 @@
+/*
```

Comment:
While I don't disagree, I can potentially see future problems arising when I try to introduce namespacing via modules.  
Additionally, Swift 2.x won't be supported by Apple any longer after 2.x next year, so that might make a good case to axe the old one and just call this the canonical `Swift` generator, thoughts?